common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -149,9 +153,19 @@ def make_selection(cls, selected, children, max_count, mode):
             'added' (set) of newly added (block_type, block_id) tuples
         """
         selected = set(tuple(k) for k in selected)  # set of (block_type, block_id) tuples assigned to this student
-
+        log_id = uuid4()
+        logger.warning(
+            "EDUCATOR-1290: LibraryContentModule.make_selection executing. selected: {0} | log ID: {1}".format(
+                selected,
+                log_id
+            )
+        )
         # Determine which of our children we will show:
         valid_block_keys = set([(c.block_type, c.block_id) for c in children])
+        "EDUCATOR-1290: LibraryContentModule.make_selection executing. valid_block_keys: {0} | log ID: {1}".format(
```

Comment:
This is missing a log.warning call around it.

Reply:
LOL